index.js

```diff
@@ -14,13 +14,18 @@ const {
     const region = core.getInput('region')
     const groupName = core.getInput('group_name')
     const newTaskDefinitionArn = core.getInput('new_task_definition_arn')
+    const exceptions = core.getMultilineInput('exceptions') || [];
```

Comment:
The `getMultilineInput` allows us to get the string list of exceptions that is separated by newline.